src/account-balance/agent.js

```diff
@@ -47,13 +47,23 @@ function createAlert(
   return Finding.fromObject(findingObject);
 }
 
+const validateConfig = (config) => {
+  let ok = false;
+  let errMsg = "";
+
```

Comment:
There's no check for the normal fields that all bots need:
```suggestion
  if (config["developerAbbreviation"] === undefined) {
      errMsg = `No developerAbbreviation found`;
      return { ok, errMsg };
  }
  if (config["protocolName"] === undefined) {
      errMsg = `No protocolName found`;
      return { ok, errMsg };
  }
  if (config["protocolAbbreviation"] === undefined) {
      errMsg = `No protocolAbbreviation found`;
      return { ok, errMsg };
  }
```